### PR TITLE
update comment about concurrency

### PR DESCRIPTION
**Pull request by wddgit**
#### PR description:

Update comments about concurrency. This only affects comments. None of the executable code is changed

#### PR validation:

Nothing should change.

### CODE REVIEW

**Comment by wddgit** [comment]
please test

FYI @makortel @Dr15Jones

**Comment by tlampen** [comment]
+1

File: CondCore/ESSources/plugins/CondDBESSource.h [hunk 1]
```diff
@@ -11,25 +11,51 @@
...
+//   near the beginning of ::DataProxy::prefetch so that these functions
+//   will never run concurrently. All the ::DataProxy objects have a
+//   pointer to this mutex stored in their ESSourceDataProxyTemplate
+//   base class.
 //
-//   2. Calls are sequenced that a call to setIntervalFor is made, then
-//   all related calls to DataProxy::initializeForNewIOV are made before
-//   another call to setIntervalFor is made.  It is configurable how many
+//   2. CondDBESSource contains a single SerialTaskQueue. The tasks
+//   that run the prefetch function are placed this SerialTaskQueue.
```
Comment (Dr15Jones) [inline CondCore/ESSources/plugins/CondDBESSource.h:27]:
```suggestion
//   that run the prefetch function are placed in this SerialTaskQueue.
```

**Comment by wddgit** [comment]
please test

**Comment by pohsun** [comment]
+1

**Comment by ggovi** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**